sig/datadog/core/environment/ext.rbs

```diff
@@ -2,21 +2,23 @@ module Datadog
   module Core
```

Comment:
Thanks for updating the signatures! But do remember to drop the `ignore` from the `Steepfile` otherwise this isn't really getting checked :)

(Same for other files that you touched :P )

Reply:
I wasn't aware that this is ignored... I dropped all ignores from Telemetry signatures a while ago, will do it for this file too